src/components/home/What.jsx

```diff
@@ -0,0 +1,20 @@
+import Title from "@/components/Title";
+import Button from "@/components/Button";
+
+const What = () => {
+  return (
+    <div className="flex flex-col items-center justify-center h-screen">
```

Comment:
component doesn't need to take the entire page.
```suggestion
    <div className="flex flex-col items-center justify-center">
```